docs/source/tutorial_mturk.rst

```diff
@@ -156,7 +158,7 @@ Sometimes you may find that a task you have created is leading to a lot of worke
 Reviewing Turker's Work
 -----------------------
 
-After all HITs are completed, you can review the work through Amazon's online interface. You can also programmatically review work using the commands available in the `MTurkManager` class.
+After all HITs are completed, you can review the work through Amazon's online interface. You can also programmatically review work using the commands available in the `MTurkManager` class. See, for example, the  `review_work function <https://github.com/facebookresearch/ParlAI/blob/master/parlai/mturk/tasks/personachat/personachat_collect_personas/worlds.py/>`__ in the ``personachat_collect_personas`` task. In this task, HITs are automatically approved if they are deemed completed by the world.
```

Comment:
Can we remove the part about reviewing work through Amazon's online interface? They no longer support that :(